patches/api/0320-Missing-Entity-Behavior-API.patch

```diff
@@ -4,6 +4,45 @@ Date: Fri, 28 May 2021 21:06:59 -0400
 Subject: [PATCH] Missing Entity Behavior API
 
 
+diff --git a/src/main/java/org/bukkit/entity/Cat.java b/src/main/java/org/bukkit/entity/Cat.java
+index c2a566b864c82ffb094b7334d9e6e25a1bfc87d1..04bdc808ee5778c56dc2f722e4f7a8e932380b44 100644
+--- a/src/main/java/org/bukkit/entity/Cat.java
++++ b/src/main/java/org/bukkit/entity/Cat.java
+@@ -54,4 +54,34 @@ public interface Cat extends Tameable, Sittable {
+         JELLIE,
+         ALL_BLACK;
+     }
++
++    // Paper Start - More cat api
++    /**
++     * Sets if the cat is resting.
++     *
++     * @param resting is resting
++     */
++    public void setIsResting(boolean resting);
```

Comment:
idk, I've never liked the setIsResting pattern. Doesn't setResting mean the same, and sound way better? Or it looks like the NMS method is called setLying, that works too. No reason it can't be the same I don't think.